app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,17 @@
+class CommentsController < ApplicationController
+  def create
+    @post = Post.find(params[:id])
+    @comment = @post.comments.new(comment_params)
+    @comment.user = current_user
+
+    return unless @comment.save
+
+    redirect_to user_post_path(current_user, @post)
+  end
```

Comment:
- [ ] Hi @abdozayan12, I was hit with the console error below when I tried to create a comment for a post. I think it will be nice to fix the error. This will allows a user to create a comment for a post successfully :+1: 
![Screenshot from 2023-10-05 06-58-00](https://github.com/abdozayan12/DataApp/assets/63087491/50be1040-c467-4f21-8b4f-a3eb809de9b9)